ui/node/attributes.go

```diff
@@ -9,16 +9,29 @@ const (
 	InputAttributeTypeCheckbox      InputAttributeType = "checkbox"
 	InputAttributeTypeHidden        InputAttributeType = "hidden"
 	InputAttributeTypeEmail         InputAttributeType = "email"
+	InputAttributeTypeTel           InputAttributeType = "tel"
 	InputAttributeTypeSubmit        InputAttributeType = "submit"
 	InputAttributeTypeButton        InputAttributeType = "button"
 	InputAttributeTypeDateTimeLocal InputAttributeType = "datetime-local"
 	InputAttributeTypeDate          InputAttributeType = "date"
 	InputAttributeTypeURI           InputAttributeType = "url"
 )
 
+const (
```

Comment:
You will need to define these in the swagger spec manually, similar to

https://github.com/ory/kratos/blob/037c0957cca2d6695b83b564efeaf17105d99b19/.schema/openapi/patches/identity.yaml#L6